packages/broker/src/plugins/operator/ContractFacade.ts

```diff
@@ -171,6 +182,41 @@ export class ContractFacade {
         return results
     }
 
+    async getExpiredRelevantFlags(sponsorships: EthereumAddress[], maxFlagAgeSec: number): Promise<Flag[]> {
+        const maxFlagStartTime = Math.floor(Date.now() / 1000) - maxFlagAgeSec
+        const createQuery = (lastId: string, pageSize: number) => {
+            return {
+                query: `
+                {
+                    flags (where : {
+                        id_gt: "${lastId}",
+                        flaggingTimestamp_lt: ${maxFlagStartTime},
+                        result_in: ["waiting", "voting"],
+                        sponsorship_in: ${JSON.stringify(sponsorships)}
+                    }, first: ${pageSize}) {
+                        id
+                        flaggingTimestamp
+                        target {
+                            id
+                        }
+                        sponsorship {
+                            id
+                        }
+                    }
+                }`
+            }
+        }
+        const parseItems = (response: { flags: Flag[] }): Flag[] => {
+            return response.flags ?? []
+        }
+        const flagsResults = this.theGraphClient.queryEntities<Flag>(createQuery, parseItems)
```

Comment:
Maybe rename as `result` has a explicit meaning in `Flag` (the status of the flag).

```suggestion
        const entities = this.theGraphClient.queryEntities<Flag>(createQuery, parseItems)
```